src/app/content/components/BookBanner.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import withServices from '../../context/Services';
+import styled from 'styled-components';
+import theme from '../../theme';
+import { AppServices, AppState } from '../../types';
+import * as select from '../selectors';
+import { Book, Page } from '../types';
+import {ChevronLeft} from 'styled-icons/boxicons-regular/ChevronLeft'
+import { findArchiveTreeSection } from '../utils'
+import Color from 'color';
+import typography from '../../components/Typography';
+
+const LeftArrow = styled(ChevronLeft)`
+  height: 2rem;
+  width: 2rem;
+  color: #FFFFFF;
+  margin-right: 7px;
+`;
+
+interface PropTypes {
+  page?: Page;
+  book?: Book;
+  services: AppServices;
+}
+
+const TopBar = styled.div`
+  height: 13rem;
+  width: 100%;
+  max-width: 1170px;
+  font-size: 1.25rem;
+  font-weight: bold;
+  background: transparent;
+  margin: 0 auto;
+  text-align: left;
+  padding-top: 1.4rem;
+`;
+
+const BookTitle = styled.h4`
+  letter-spacing: -0.4;
+  color: #FFFFFF;
+  text-align: left;
+  display: inline-block;
+  width: 100%;
+  max-width: 87rem;
+  margin: 1.25rem 0;
+  line-height: 1.5rem;
+`;
+
+const BookChapter = styled.h3`
+  letter-spacing: -0.04rem;
+  color: #FFFFFF;
```

Comment:
this color is going to depend on the background color, some of them, like maybe the yellow ones, will want darker text. in the `theme` i put a `foreground` on some of the color sets to keep track of what text color things should use, we should add one to blue and use it here